support-frontend/assets/components/socialShare/socialShare.scss

```diff
@@ -1,10 +1,11 @@
 .component-social-share {
 	svg {
 		width: 20px;
-		border: 1px solid gu-colour(neutral-7);
+		border: 1px solid gu-colour(neutral-86);
 		border-radius: 600px;
 		padding: 8px;
 		// Fix for IE10.
```

Comment:
We don't support IE10 technically so if this is only a visual issue (it works but looks a bit ugly) I'd personally leave it ugly rather than adding this fix because it makes the component harder to maintain

Reply:
Yeah this is just legacy code, I can delete